spec/controllers/concerns/third_party_transaction_logging_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ThirdPartyTransactionLogging do
+  let(:test_controller_class) do
```

Comment:
our linter complains about defining a class inside of a block.  RSpec does not provide a meaningful way to test a concern (or any module) in isolation.  I believe this module is worth testing in isolation.  Testing its use in an actual controller would be its own sort of anti-pattern.   Alternatively, we could get past this linter complaint by defining this TestController outside of the `Rspec.descrbe` block, but then it would pollute the global name space.